recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_%.bbappend

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+do_install_append () {
+    rm ${D}${sysconfdir}/${PN}/config.conf
+    install -d ${D}/mnt/config/hawkbit
+    install -m 644 ${S}/config.conf.example ${D}/mnt/config/hawkbit/config.cfg
```

Comment:
This can be removed, as the actual configuration file is provided by the awsclient from ESEC's IoT Device Manager platform. You only need to create the symbolic link.